spec/controllers/sessions_controller_spec.rb

```diff
@@ -18,14 +18,35 @@
       expect(response).to redirect_to(posts_url)
     end
 
-    it 'unable to login with invalid details' do
-      expect {
-        post :create,
-             params: {
-               email: 'hilly@example.com',
-               password: 'griltheAnim4lz'
-             }
-      }.to raise_error
+    it 'responds 200 when login details are wrong' do
+      post :create, params: { email: '', password: '' }
+      expect(response).to have_http_status(302)
+    end
+
+    it 'alerts user if wrong password or email given' do
+      post :create, params: { email: '', password: '' }
+      expect(flash[:alert]).to match('Invalid email or password')
```

Comment:
Combine with above? 
aka 
expect(response).to have_http_status(302)
expect(flash[:alert]).to match('Invalid email or password')

As same setup is used for both tests.